src/main/java/edu/ie3/datamodel/utils/validation/ConnectorValidationUtils.java

```diff
@@ -193,7 +191,7 @@ private static void checkTransformer3W(Transformer3WInput transformer3W) {
    * - rScA, rScB, rScC are greater 0 (short circuit resistance in branches A,B,C) <br>
    * - xScA, xScB, xScC are greater 0 (short circuit impedance in branches A,B,C) <br>
    * - gM is greater/equal to 0 (no load conductance) <br>
-   * - bM is greater/equal to 0 (no load susceptance) <br>
+   * - bM is less/equal to 0 <br>
```

Comment:
```suggestion
   * - bM is greater/equal to 0 (no load susceptance) <br>
```

Reply:
bM should be negative so shouldn't it be **less/equal to 0**?

Reply:
You are right! Sorry, my bad!